packages/mobile/src/components/app-navigator/TopBar.tsx

```diff
@@ -117,7 +122,7 @@ export const TopBar = memo(
               />
             )}
           </View>
-          {title ? (
+          {title === null ? null : title ? (
```

Comment:
I'd prefer if we didn't use a double ternary, this seems confusing

Reply:
yeah agreed as a general rule, been trying it out a lil more recently, but yeah is confusing